autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -55,6 +55,22 @@ const SectionSelect: React.FC<SectionSelectProps> = ({ id }): JSX.Element => {
       - ${formatTime(mtg.endTimeHours, mtg.endTimeMinutes)}`;
   };
 
+  /**
+   * Accepts an array of meetings and returns a filtered array without duplicate meetings.
+   * Meetings are considered to be duplicates if they are of the same type, meet on the same days,
+   * and start and end at the same hour. Meetings that are the same by all of these criteria but
+   * differ only in the exact minutes of the meeting will still be considered duplicates
```

Comment:
I'm not sure exactly how I feel about this since it only depends on hours and will end up being inaccurate in some cases (for example if there are 2 exams, one is 8:00-9:30 and the other is 8:00-10:00, they won't be considered the same, if there is instead one from 8:00-9:30 and another from 8:00-9:00, the time will show as 8:00-9:00 despite one of the meetings going later), I think a better way of doing this (although it'll make your pretty clever code a little more verbose) would be making the start time in hours and minutes the key, and when going through meetings if the key already exists, update the end time to the later of the existing value and the meeting being added

Reply:
Makes sense, but the algorithm you described would miss the case where one of the exams had a slightly different start time than the others, if it were ever to occur. That's fine by me, but I just wanted to double check before implementing it.

Reply:
As per meeting on 6/14, I have implemented the requested behavior